config/application.rb

```diff
@@ -35,11 +35,11 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :sidekiq
 
-    config.lograge.enabled = true
-    config.lograge.enabled = Lograge::Formatters::Json.new
-    config.log_formatter = proc do |severity, datetime, progname, msg|
-      "time=\"#{datetime.iso8601(3)}\" level=\"#{severity}\" pid=\"#{Process.pid}\" #{msg.to_s.gsub('"', "'")}\n"
-    end
+    # config.lograge.enabled = true
+    # config.lograge.enabled = Lograge::Formatters::Json.new
+    # config.log_formatter = proc do |severity, datetime, progname, msg|
+    #   "time=\"#{datetime.iso8601(3)}\" level=\"#{severity}\" pid=\"#{Process.pid}\" #{msg.to_s.gsub('"', "'")}\n"
+    # end
```

Comment:
Sorry @steveyken , pushed wrong file. But yeah it's a hard read. But we can work with comments in dev mode. Do we want to push it to staging and prod also?
If so then I'll keep this otherwise I'll revert this commit.

Let me know